pam/integration-tests/testdata/tapes/mfa_auth.tape

```diff
@@ -0,0 +1,81 @@
+Output mfa_auth.txt
+Output mfa_auth.gif # If we don't specify a .gif output, it will create a default out.gif file.
+
+# Configuration header to standardize the output.
+# Does not work with the "Source" command.
+Set Width 1800
+Set Height 800
```

Comment:
I'd reduce this by A LOT.

Terminals can be smaller, but more than that we avoid lots of clutter in the golden files so that they're not full of empty lines or spaces.

IMHO It's more than enough using `300x200` or similar. If something needs to be wrapped, bubbletea will do it.

In any case we should also have tests with even tinier terminal, just to ensure that everything work in edge cases.

Reply:
`vhs` has a minimum requirement for the terminal size, I believe it's 400x400 IIRC. When using small windows, we can't see if the values returned by PAM are the correct ones, since some of the lists (the authentication modes list, for example) get cut in half. I think it's more important (in our case) to test that the communication between Client <-> PAM <-> Daemon <-> Broker is correct, rather than the formatting on the terminal itself (which is likely covered in the bubble tests.) In any case, we will need to so some manual testing to watch for stuff like "behavior when resizing the terminal during the auth process".